app/main.py

```diff
@@ -2,0 +1,15 @@
+def calculate_team_total_rating(players: list) -> int:
+    player_reting = []
```

Comment:
There is a typo in the variable name `player_reting`. It should be `player_rating` if you intend to use it.